tests/event_reader.rs

```diff
@@ -355,13 +363,6 @@ fn test(input: &[u8], output: &[u8], config: ParserConfig, test_position: bool)
     // to stderr instead of comparing with the output
     // it can be used like this:
     // PRINT_SPEC=1 cargo test --test event_reader sample_1_full 2> sample_1_full.txt
-    START.call_once(|| {
```

Comment:
These lines contained comments which describe the logic of the `PRINT` flag. Please move them to the appropriate place, as well.